paramak/reactor.py

```diff
@@ -270,7 +270,8 @@ def export_neutronics_description(
 
         return str(path_filename)
 
-    def export_stp(self, output_folder="", graveyard_offset=100):
+    def export_stp(self, output_folder="", graveyard_offset=100,
+                   solid_or_wire='solid'):
```

Comment:
```suggestion
                   mode='solid'):
```

I think solid_or_wire is a bit heavy. Could something like "mode" do the trick ?